evap/grades/views.py

```diff
@@ -117,29 +117,19 @@ def upload_grades(request, semester_id, course_id):
 
 
 @grade_publisher_required
-def toggle_no_grades(request, semester_id, course_id):
-    semester = get_object_or_404(Semester, id=semester_id)
+def toggle_no_grades(request):
```

Comment:
i just found https://docs.djangoproject.com/en/1.9/topics/http/decorators/#django.views.decorators.http.require_POST, could you add that to all our views that are only for POST requests?